qchem/pennylane_qchem/qchem/hamiltonian.py

```diff
@@ -214,8 +215,9 @@ def _molecular_hamiltonian(*args):
         Returns:
             Hamiltonian: the qubit Hamiltonian
         """
-        h_ferm = fermionic_hamiltonian(mol, cutoff, core, active)(*args)
-
-        return qubit_observable(h_ferm)
+        with stop_recording():
+            h_ferm = fermionic_hamiltonian(mol, cutoff, core, active)(*args)
+            out = qubit_observable(h_ferm)
```

Comment:
Why do you need to manually stop recording here? Is there anything inside the fermionic hamiltonian that queues?

Reply:
something inside `qubit_observable` I think.  Honestly had trouble figuring out where the terms where coming from, so that's why I just wrapped everything in the `stop_recording` context.